README.adoc

```diff
@@ -1,100 +1,67 @@
+It receives metrics from the Heapster component and is also used to expose metrics to the console and other third party systems.
 
-== Building the Docker Containers
+==== Cassandra
+link:http://cassandra.apache.org/[Cassandra] is the data store for Hawkular Metrics. It stores and persists all the metrics coming in from Hawkular Metrics.
 
-All the docker images for Origin Metric components are available at link:https://hub.docker.com/search/?q=openshift%2Forigin-metrics&page=1&isAutomated=0&isOfficial=0&starCount=0&pullCount=0[docker hub] and there should not be a need to build these directly.
+==== Hawkular OpenShift Agent
+The link:https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift Agent] is a component which can be used to gather application level metrics coming from pods. This allows pods themselves to expose metrics that they wish to be collected.
 
-If you wish to build your own images or hack on the project. Please see the link:docs/build.adoc[build instructions].
+This component runs as a deamon set across the OpenShift cluster and is responsible for gathering application level metrics from each pod running on that node.
+
+[NOTE]
+====
+The Hawkular OpenShift Agent is currently in Tech Preview.
+====
 
 == Before You Begin
 
 The Metrics components requires that OpenShift is properly installed and configured. How to properly install and configure OpenShift is beyond the scope of this document. Please see the link:https://docs.openshift.org/latest/welcome/index.html[OpenShift documentation] on how to properly install and setup your system.
 
-There are a few things which you will specifically need to make sure are properly configured with your OpenShift instance:
+The metrics install requires integation with the OpenShift cluster and if there are installation or configuration problems with your OpenShift cluster you may encounter them when tyring to running metrics. Some of the more common issues are checked in the deployer pod and will give you an error message about your installation.
 
-. The link:docs/troubleshooting.adoc#checking-if-the-dns-service-is-running-or-not[OpenShift DNS server] needs to be properly running on your system.
+Please see the link:docs/troubleshooting.adoc[troubleshooting guide] for a complete list of things to watch out for.
 
-. link:docs/troubleshooting.adoc#empty-charts[Node certificate problems] if your OpenShift installation was originally installed on an older version.
+== Installing
 
-. SELinux and firewall configurations may cause problems if not properly configured.
+The link:https://docs.openshift.org/latest/install_config/cluster_metrics.html[OpenShift documentation] describes how to install metrics.
 
-Please see the link:docs/troubleshooting.adoc[troubleshooting guide] for a complete list of things to watch out for.
-
-== Deploying the Origin Metrics Components
+A brief overview of the steps required are as follows:
```

Comment:
@jcantril: remove this whole section and just have people go to the main docs? or what are you thoughts?